redfish-core/lib/assembly.hpp

```diff
@@ -540,6 +620,39 @@ inline void setAssemblylocationIndicators(
                         "xyz.openbmc_project.adcsensor.service", "replace");
                 }
             }
+            // Special handling for LCD and base panel CM.
+            else if (sdbusplus::message::object_path(assembly).filename() ==
+                         "panel1" ||
+                     sdbusplus::message::object_path(assembly).filename() ==
+                         "panel0")
+            {
+                std::string action{};
+                if (readytoremove.value() == true)
+                {
+                    // Set action for removal of FRU.
+                    action = "deleteFRUVPD";
```

Comment:
Why does a readyToRemove mean a delete of the FRU VPD?

Reply:
"readyToRemove" here act as a trigger point for removal of FRU. Once it is set to "true" in GUI implies the FRU should be prepared for removal. 
So in order to remove the FRU we need to unbind the driver attached to the FRU and set its presence to "false". So the api which collectively does steps required to remove the FRU is exposed as "deleteFRUVPD" via vpd-manager.

Reply:
This is also the same pair of APIs that PHYP uses to indicate a CM operation for PCIe cards.

Reply:
Can we add a comment here explaining this, this is just unexpected

Reply:
Done